docs/docs/how-tos/domain-registry.md

```diff
@@ -6,12 +6,13 @@ description: Setting a custom DNS Domain Registry
 
 # Nebari domain registry setup
 
-This domain will be where your application will be exposed.
+Take IP Address generated for the ingress from the terminal and set up the DNS. Setting a DNS record heavily depends on the provider. Below is the links of detailed docs of DNS providers on how to create a DNS record on all the providers out there. 
+
+1. [Cloud DNS](https://cloud.google.com/dns/docs/tutorials/create-domain-tutorial) provider.
+2. [Amazon Route 53](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/domain-register.html) DNS provider.
+3. [Azure DNS](https://docs.microsoft.com/en-us/azure/dns/dns-getstarted-portal) provider.
+4. [Digital Ocean DNS](https://docs.digitalocean.com/products/networking/dns/quickstart/) provider.
 
-Currently, Nebari's default configuration is set for CloudFlare for automatic DNS registration. If you prefer an alternate DNS provider, you'll need to change the `--dns-provider` flag from `cloudflare` to `none` on the QHub deploy command.
-The deployment process will then pause when it asks for an IP address (or `CNAME`, if using AWS) and prompt you to provide the desired URL.
+This domain will be where your application will be exposed.
 
-:::warning
-Setting a DNS record heavily depends on the provider, and thus it’s not possible to provide detailed docs here on how to
-create a record on all the providers out there. Searching "setting a DNS record" or "setting a CNAME DNS record" on your specific provider should yield good results.
-:::
+When you are done with setting up the domain name. You can switch back to the previous page and continue the further steps to deploy Nebari. 
```

Comment:
:::note
Once your new DNS domain is created, it might take time for the records and related certs to fully propagate over the internet.

The amount of time you might need to wait varies from each DNS provider. Validate such information in the related documentation for your chosen provider.
:::

### Automatically DNS provision

Qhub has an extra flag for deployments that grants management and creation of the DNS records for you automatically,  for automatically DNS provision add `--dns-auto-provision` to your `qhub deploy` command:

```
qhub deploy -c qhub-config --dns-provider cloudflare --dns-auto-provision
```
This will set the DNS provider as Cloudflare and automatically handle the creation or updates for Qhub domain DNS records on Cloudflare.

:::warn
The usage of `--dns-auto-provision` is restricted to Cloudflare as it is the only fully integrated DNS provider that Qhub currently supports.
:::

When you are done setting up the domain name, you can switch back to the Nebari deployment documentation and further continue the remaining steps.